### PR TITLE
Energy fix

### PR DESCRIPTION
**Pull request by Nick-Chaos**
Corrected energy in QMMM calculations and added a test for QM with puntual charges.

tested with intel and gfortran

### CODE REVIEW

File: lioamber/dft_get_mm_forces.f [hunk 1]
```diff
@@ -93,6 +93,7 @@ subroutine dft_get_mm_forces(dxyzcl,dxyzqm)
        call g2g_timer_summary()
        call g2g_timer_clear()
 
+
```
Comment (fedepedron) [inline lioamber/dft_get_mm_forces.f]: Really? Jajajajaja

File: test/14_QM_in_Pcharges/basis [hunk 2]
```diff
@@ -0,0 +1,52 @@
+gaussian
```
Comment (fedepedron) [inline test/14_QM_in_Pcharges/basis]: Is this file really necessary? Shouldn't be easier just putting int_basis=t? or is it a custom basis?

Comment (Nick-Chaos) [inline test/14_QM_in_Pcharges/basis]: yes, i just copy 0_aguaGPU test and added point of charges

File: lioamber/dft_get_mm_forces.f (hunk 1 above)
Comment (Nick-Chaos) [inline lioamber/dft_get_mm_forces.f]: obiamente el cambio mas importante
